.tekton/integration-tests/lightspeed-console-pre-commit.yaml

```diff
@@ -232,7 +232,7 @@ spec:
               echo "---------------------------------------------"
               echo "npm version: $(npm -v)"
               echo "---------------------------------------------"
-              NODE_OPTIONS=--max-old-space-size=4096 npm ci --omit=optional --no-fund
+              NODE_OPTIONS=--max-old-space-size=4096 npm install npm ci --omit=optional --no-fund
```

Comment:
This will actually try to install the `npm` and `ci` packages and it just so happens that both exist 😆

Just running `npm ci` is normally best practice and `npm install` shouldn't be necessary, so let's try to figure out why `npm ci` is erroring.